### DISCUSSION TITLE
可以播放3D原盘吗

### DISCUSSION THREAD

**Discussion by bulesky89857** [hooke007/mpv_PlayKit#282] [Q&A 问答]
请问如何播放3D原盘文件？
mpv的补帧是真的很好使，想播放3D原盘，+补帧。
播放原盘里面的m2ts文件 是2D画面，如何切换到3D输出呢？

**Comment by s3327445** [comment]
可以 資料夾拖過去就可以了

**Reply by bulesky89857** [reply]
> 可以 資料夾拖過去就可以了
无效

**Comment by hooke007** [comment]
暂不支持输出3D画面 https://github.com/mpv-player/mpv/issues/2583
